src/pages/base/markets/components/RateHistoryChart.js

```diff
@@ -46,11 +49,13 @@ function RateHistoryChart(props) {
       label: "borrow APY",
       data: borrow,
     },
-    {
-      label: "borrow stable APY",
-      data: stable_borrow_apy,
-    },
   ];
+  if (data[0].stable_borrow_rate_enabled === true) {
```

Comment:
This is very flaky as it might not be the first one that's enabled, but could be the second one.
Better way to check for this would be to check `if (stable_borrow_apy.length > 0)` as you won't add anything to `stable_borrow_apy` if all rows are not borrow rate enabled